extensions/railway/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "railway",
+  "title": "Railway",
```

How about naming it like this? Might look better in the store. If you ad more commands, we can rename the extension easily.

```suggestion
  "title": "Railway Project Search",
```